README.md

```diff
@@ -1,105 +1,149 @@
+| [MiniWDL](https://github.com/chanzuckerberg/miniwdl)                           | Yes                   | Yes             | SLURM | AWS Batch             |
+| [Terra](https://terra.bio/)                                                    | No                    | No              | No    | Azure, GCP            |
+
+\* Also see [WDL Runner](https://github.com/broadinstitute/wdl-runner), a script for launch WDL workflows on GCP using Cromwell
+
+### Grammars, Parsers, and Language Support
+
+- The WDL [parsers repository](https://github.com/openwdl/wdl-parsers/) provides grammar definitions in various formats and generated parsers for various languages.
+- [MiniWDL](https://github.com/chanzuckerberg/miniwdl) provides python bindings for WDL as well as commandline validation and linting tools.
+- [WOMTool](https://cromwell.readthedocs.io/en/stable/WOMtool/) is a standalone Java tool for WDL parsing, validating, linting, and diagramming.
+- [wdlTools](https://github.com/dnanexus/wdlTools) - provides 1) a parser Java/Scala library, based on the  [ANTLR4 grammars](https://github.com/openwdl/wdl-parsers) grammars, for WDL draft-2, 1.0, 1.1, and 2.0; and 2) command-line tools for sytanx checking, type-checking, linting, code formatting (including upgrading from older to newer WDL versions), generating documentation, and executing WDL tasks locally.
 
 ### IDE Support
 
-- Visual Studio Code: [WDL Syntax Highlighter](https://marketplace.visualstudio.com/items?itemName=broadinstitute.wdl)
-- JetBrains IDE's: [Winstanly](https://plugins.jetbrains.com/plugin/8154-winstanley-wdl)
-- Atom: [Language-WDL](https://atom.io/packages/language-wdl)
-- Vim: [vim-wdl](https://github.com/broadinstitute/vim-wdl)
+| IDE                | Tool                                                                                             |
+| ------------------ | ------------------------------------------------------------------------------------------------ |
+| Atom               | [Language-WDL](https://atom.io/packages/language-wdl)                                            |
+| Emacs              | [poly-wdl](https://github.com/jmonlong/poly-wdl)                                                 |
+| Emacs              | [wdl-mode](https://github.com/zhanxw/wdl-mode)                                                   |
+| JetBrains          | [Winstanly](https://plugins.jetbrains.com/plugin/8154-winstanley-wdl)                            |
+| Sublime            | [WDL Syntax Highlighter](https://github.com/broadinstitute/wdl-sublime-syntax-highlighter)       |
+| Vim                | [vim-wdl](https://github.com/broadinstitute/vim-wdl)                                             |
+| Visual Studio Code | [WDL Syntax Highlighter](https://marketplace.visualstudio.com/items?itemName=broadinstitute.wdl) |
 
 ### Documentation
```

Comment:
Should you add mini-wdl and and the input_template: https://miniwdl.readthedocs.io/en/latest/input_template.html

Reply:
Not sure that belongs in IDE support - I mentioned it under language support instead